project_national/fact_sheets_category_6.yml

```diff
@@ -0,0 +1,979 @@
+
+workflow_generator:
+  type: residential_hpxml
+  args:
+    build_existing_model:
+      simulation_control_timestep: 60
+      simulation_control_run_period_begin_month: 1
+      simulation_control_run_period_begin_day_of_month: 1
+      simulation_control_run_period_end_month: 12
+      simulation_control_run_period_end_day_of_month: 31
+      simulation_control_run_period_calendar_year: 2007
+
+    emissions:
+      - scenario_name: LRMER_HighRECost_2030boxavg
+        type: CO2e
+        elec_folder: data/cambium/2022/LRMER_HighRECost_2030boxavg
+        gas_value: 147.3
+        propane_value: 177.8
+        oil_value: 195.9
+
+      - scenario_name: LRMER_LowRECost_2030boxavg
+        type: CO2e
+        elec_folder: data/cambium/2022/LRMER_LowRECost_2030boxavg
+        gas_value: 147.3
+        propane_value: 177.8
+        oil_value: 195.9
+
+      - scenario_name: LRMER_MidCase_2030boxavg
+        type: CO2e
+        elec_folder: data/cambium/2022/LRMER_MidCase_2030boxavg
+        gas_value: 147.3
+        propane_value: 177.8
+        oil_value: 195.9
+
+    utility_bills:
+      - scenario_name: Utility Rates - Fixed + Variable
+        simple_filepath: data/simple_rates/State.tsv
+
+
+    simulation_output_report:
+      timeseries_frequency: timestep #monthly
+      include_timeseries_total_consumptions: true
+      include_timeseries_fuel_consumptions: true
+      include_timeseries_end_use_consumptions: true
+      include_timeseries_emissions: true
+      include_timeseries_total_loads: false
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: false
+      retain_schedules_csv: false
+
+
+### -- Upgrades ---
+# Category 6:  Heat Pumps with Existing Backup
+# To do: costing interpolation from Less for 9.2, 9.5, and 11 HSPF
+upgrades:
+  - upgrade_name: EnergyStar ducted ASHP w/ existing natural gas backup
+    # Measure 1
+
+    package_apply_logic: &dualfuel_ASHP_apply_logic_NG
+      - HVAC Has Ducts|Yes
+      - Heating Fuel|Natural Gas
+      - not:
+          or:
+            - HVAC Heating Efficiency|Shared Heating
+            - HVAC Cooling Efficiency|Shared Cooling  
+
+    options:
+      #ASHP
+      - &ducted_ASHP_EnergyStar_exist_backup_no_shared_ducts_NG
+        option: HVAC Heating Efficiency|ASHP, SEER 16, 9.2 HSPF, Separate Backup
+        apply_logic: &dualfuel_ASHP_no_ducted_furnace_apply_logic_NG
+          - not: HVAC Heating Type And Fuel|Natural Gas Fuel Furnace
+        costs: &costs_ashp_ducted_SEER_16_9-2_HSPF
+        # Source: Custom regression by Brennan Less. Ducted heat pump project costs were regressed on nameplate tons and HSPF,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. January 2019 $ inflated to April 2023 $ using 1.21 factor.
+        # Ducted ASHP regression n=317. Implicitly includes electrical upgrades and electric backup.
+          - value: 4821.83 # for HSPF 9.2
```

Comment:
These costs appear to be the same as the costs in yaml 5, do we want that? Two thoughts:

Here we are not needing electric_backup as a separate system or a HP with an integrated electric backup, so would the equipment cost be lower because of that?

Additionally, the Chicago project has shown that removing existing heating equipment can be costly, e.g., boiler lines, wall furnaces, because existing systems tend to be bulky and requires holes to be filled. Do we want to add those to the costs in yaml 5?

Reply:
Deferring to @ekpresent for this

Reply:
@trynthink elevated this and the conclusion is to not add any further dimensions to our HP costs for the run, so keep the costs the same regardless of backup. This is due to there being many different factors that impact HP cost and us not having data to support this one being more impactful than others that we are currently omitting.